src/main/java/apoc/ApocConfig.java

```diff
@@ -46,6 +46,7 @@
 
     public static final String SUN_JAVA_COMMAND = "sun.java.command";
     public static final Pattern CONF_DIR_PATTERN = Pattern.compile("--config-dir=(\\S+)");
```

Comment:
That line can be removed when we get rid of regex.

Reply:
done

Reply:
Thanks